openfl-workspace/torch_cnn_mnist/README.md

```diff
@@ -0,0 +1,162 @@
+## Directory Structure
+The taskrunner workspace has the following file structure:
```

Comment:
As suggested offline by @kta-intel, add a section (also present in the blog post) on how to instantiate a workspace from `torch_cnn_mnist` as a template.

Reply:
resolved in d8dda083b7c32d0680ef2d87564c3fa64b888f19